distribution/docs/src/main/resources/_contents/_platform-contents/managing-platform-contents.adoc

```diff
@@ -58,3 +58,16 @@ The Platform application is installed by default on a standard installation.
 |
 
 |===
+
```

Comment:
@ricklarsen, can you take a quick look to make sure the description and formatting fit with the rest of the docs?